app/models/vaccine.rb

```diff
@@ -25,4 +25,13 @@ def self.overbooking_factor(vaccine)
       2
     end
   end
+
+  def self.average_sms_count_cost_per_dose(vaccine)
+    case vaccine
+    when Brands::ASTRAZENECA
```

Comment:
Ici je mettrais le module `Vaccine` dans le nom, si on refactor en cherchant `Vaccine::Brands` on ne verra pas cette ligne
```suggestion
    when Vaccine::Brands::ASTRAZENECA
```